languages/python/sqlalchemy-oso/sqlalchemy_oso/enforcer.py

```diff
@@ -0,0 +1,50 @@
+from oso import Enforcer
+from .oso import SQLAlchemyPolicy
+from .auth import authorize_model
+
+
+class SQLAlchemyEnforcer(Enforcer):
```

Comment:
What do you think about removing the `SQLAlchemy` prefix? I feel like the pro of removing, is that it will remove some redundancy (`from sqlalchemy_oso import SQLAlchemyEnforcer` etc). Cons is that it might be harder to know from context in a docs guide which is imported.

Reply:
It's a good question. My gut is to keep it, and encourage the use of longer-named enforcers. The cost isn't huge (they're typically imported and used once), and like you said, it makes the docs a bit clearer what to use when.

It does make me wonder -- if I wanted to use a `FlaskEnforcer` AND a `SQLAlchemyEnforcer` as a single enforcer, I wonder how I might allow that. For now I don't think it's a huge deal, but it might become something we want to support. But I think that type of thing would be also be easier to reason about if enforcers had specific names.